### ISSUE TITLE
MARWADI UNIVERSITZ

### ISSUE THREAD

**Issue by eganesh12** [eganesh12/-marwadi-university#1]
`<!DOCTYPE html>
<html lang="en">
<head>
  <title> MARWADI UNIVERSIITY </title>
  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.7.1/jquery.min.js"></script>
  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/js/bootstrap.min.js"></script>
</head>
<body>

<nav class="navbar navbar-inverse">
  <div class="container-fluid">
    <div class="navbar-header">
      <a class="navbar-brand" href="file:///C:/Users/egane/Documents/OneDrive/Desktop/html/l.html">MARWADI UNIVERSITY</a>
    </div>
    <ul class="nav navbar-nav">
      <li class="active"><a href="#">HOME</a></li>
      <li class="dropdown"><a class="dropdown-toggle" data-toggle="dropdown" href="file:///C:/Users/egane/Documents/OneDrive/Desktop/l.htmlF">CODING <span class="caret"></span></a>
        <ul class="dropdown-menu">
          <li><a href="https://www.w3schools.com/html/default.asp">HTML</a></li>
          <li><a href="https://www.w3schools.com/css/default.asp">CSS</a></li>
          <li><a href="https://www.w3schools.com/python/default.asp">PYTHON</a></li>
        </ul>
      </li>
      <li><a href="https://www.w3schools.com/cpp/default.asp">C++</a></li>
      <li><a href="M.jpg">OR CODE</a></li>
    </ul>
    <ul class="nav navbar-nav navbar-right">
      <li><a href="j.html"><span class="glyphicon glyphicon-user"></span> Sign Up</a></li>
      <li><a href="o.html"><span class="glyphicon glyphicon-log-in"></span> Login</a></li>
    </ul>
        <form class="navbar-form navbar-left" action="/action_page.php">
      <div class="form-group">
        <input type="text" class="form-control" placeholder="Search">
      </div>
      <button type="submit" class="btn btn-default">Submit</button>
    </form>
  </div>
</nav>
  </div>
</nav>
  
<div class="container">
  <style>
.scroll-left {
 height: 50px;	
 overflow: hidden;
 position: relative;
 background: yellow;
 color: orange;
 border: 1px solid orange;
}
.scroll-left p {
 position: absolute;
 width: 100%;
 height: 100%;
 margin: 0;
 line-height: 50px;
 text-align: center;
 /* Starting position */
 transform:translateX(100%);
 /* Apply animation to this element */
 animation: scroll-left 15s linear infinite;
}
/* Move it (define the animation) */
@keyframes scroll-left {
 0%   {
 transform: translateX(100%); 		
 }
 100% {
 transform: translateX(-100%); 
 }
}
</style>

<div class="scroll-left">
<p>MARWADI UNIVERSITY TOP 23RD UNIVERSITY  RAJKOT ,GUJARAT 360001</p>
</div>

  
</div>
<!DOCTYPE html>
<html>
<head>
<meta name="viewport" content="width=device-width, initial-scale=1">
<style>
body, html {
  height: 100%;
  margin: 0;
}

.bg {
  /* The image used */
  background-image: url("9.jpg");

  /* Full height */
  height: 100%; 

  /* Center and scale the image nicely */
  background-position: center;
  background-repeat: no-repeat;
  background-size: cover;
}
</style>
</head>
<body>

<div class="bg"></div>

</body>
</html>

</body>
</html>
<!DOCTYPE html>
<html lang="en">

<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Signup Form</title>

    <style>
        body {
            font-family: Arial, sans-serif;
            background-color:;
            display: flex;
            justify-content: center;
            align-items: center;
            height: 100vh;
            margin: 0;
        }

body {
 background-image: url("3.jpg");
 background-color: #cccccc;
}
        .container {
            background-color: white;
            padding: 20px;
            border-radius: 5px;
            box-shadow: 0 0 10px rgba(0, 0, 0, 0.1);
            width: 300px;
        }

        h2 {
            margin-bottom: 20px;
            text-align: center;
        }

        .form-group {
            margin-bottom: 15px;
        }

        label {
            display: block;
            margin-bottom: 5px;
        }

        input {
            width: 100%;
            padding: 8px;
            box-sizing: border-box;
            border: 1px solid #ccc;
            border-radius: 4px;
        }

        button {
            width: 100%;
            padding: 10px;
            background-color: red;
            border: none;
            color: white;
            border-radius: 4px;
            cursor: pointer;
        }

        button:hover {
            background-color: red;
        }

        #message {
            margin-top: 20px;
            text-align: center;
        }

        .error {
            color: red;
            font-size: 0.9em;
        }

        .password-hint {
            font-size: 0.9em;
            color: #666;
            margin-top: 5px;
            display: none;
        }

        .password-strength {
            margin-top: 5px;
        }

        .password-strength div {
            height: 5px;
            border-radius: 4px;
        }

        .weak {
            width: 33%;
            background-color: red;
        }

        .medium {
            width: 66%;
            background-color: orange;
        }

        .strong {
            width: 100%;
            background-color: red;
        }
    </style>
</head>

<body>
    <div class="container">
        <h2>Signup Form</h2>
        <form id="signupForm">
            <div class="form-group">
                <label for="username">Username</label>
                <input type="text" id="username" name="username" required>
            </div>
            
            <div class="form-group">
                <label for="email">Email</label>
                <input type="email" id="email" name="email" required>
            </div>
            <div class="form-group">
                <label for="password">Password</label>
                <input type="password" id="password" name="password" required>
                <div class="password-hint" id="passwordHint">
                    Password must be at least 8 characters long, and include at
                    least one uppercase letter, one lowercase letter, one
                    number, and one special character.
                </div>
                <span id="passwordError" class="error"></span>
                <div class="password-strength" id="passwordStrength"></div>
            </div>
            <div class="form-group">
                <label for="confirmPassword">Confirm Password</label>
                <input type="password" id="confirmPassword"
                    name="confirmPassword" required>
                <span id="confirmPasswordError" class="error"></span>
            </div>
            <button type="submit">Signup</button>
        </form>
        <p id="message"></p>
    </div>
    <script>
        document.getElementById('signupForm').addEventListener('submit', function (event) {
            event.preventDefault();

            let username = document.getElementById('username').value;
            let email = document.getElementById('email').value;
            let password = document.getElementById('password').value;
            let confirmPassword = document.getElementById('confirmPassword').value;
            let message = document.getElementById('message');
            let passwordError = document.getElementById('passwordError');
            let confirmPasswordError = document.getElementById('confirmPasswordError');

            let passwordValid = validatePassword(password);
            if (!passwordValid) {
                passwordError.textContent = 'Password does not meet the requirements.';
                return;
            } else {
                passwordError.textContent = '';
            }

            if (password !== confirmPassword) {
                confirmPasswordError.textContent = 'Passwords do not match!';
                return;
            } else {
                confirmPasswordError.textContent = '';
            }

            // Here you can add code to send the form data to the server

            message.style.color = 'green';
            message.textContent = 'Signup successful!';
        });

        document.getElementById('password').addEventListener('focus', function () {
            document.getElementById('passwordHint').style.display = 'block';
        });

        document.getElementById('password').addEventListener('blur', function () {
            document.getElementById('passwordHint').style.display = 'none';
        });

        document.getElementById('password').addEventListener('input', function () {
            let password = this.value;
            let passwordStrength = document.getElementById('passwordStrength');
            let strength = getPasswordStrength(password);

            passwordStrength.innerHTML = ''; // Clear previous strength bars
            if (strength) {
                let strengthBar = document.createElement('div');
                strengthBar.className = strength;
                passwordStrength.appendChild(strengthBar);
            }
        });

        function validatePassword(password) {
            let regex = /^(?=.*[a-z])(?=.*[A-Z])(?=.*\d)(?=.*[@$!%*?&])[A-Za-z\d@$!%*?&]{8,}$/;
            return regex.test(password);
        }

        function getPasswordStrength(password) {
            if (password.length < 8) {
                return 'weak';
            }
            if (password.match(/(?=.*[a-z])(?=.*[A-Z])(?=.*\d)(?=.*[@$!%*?&])/)) {
                return 'strong';
            }
            return 'medium';
        }
    </script>

</body>    

</html>
body{
  padding: 25px;
}
.title {
	color:blue;
}
/* Reset default styles and set font family */
* {
  padding: 0;
  margin: 0;
  box-sizing: border-box;
  font-family: "Poppins", sans-serif;
}

/* Center the content vertically and horizontally */
body {
  display: flex;
  justify-content: center;
  align-items: center;
  background-color: ;
  height: 100vh;
}
body {
 background-image: url("8.jpg");
 background-color: #cccccc;
}
/* Main container styles */
.box {
  position: relative;
  width: 370px;
  height: 450px;
  background: black;
  border-radius: 50px 5px;
  overflow: hidden;
}

/* Create animated gradient border effect */
.box::before,
.box::after {
  content: "";
  position: absolute;
  top: -50%;
  left: -50%;
  width: 370px;
  height: 450px;
  background: linear-gradient(60deg, transparent, #45f3ff, #45f3ff);
  transform-origin: bottom right;
  animation: animate 6s linear infinite;
}

.box::after {
  background: linear-gradient(60deg, transparent, #d9138a, #d9138a);
  animation-delay: -3s;
}

/* Keyframes for gradient animation */
@keyframes animate {
  0% {
    transform: rotate(0deg);
  }
  100% {
    transform: rotate(360deg);
  }
}

/* Form container styles */
form {
  position: absolute;
  inset: 2px;
  background: #28292d;
  border-radius: 50px 5px;
  z-index: 10;
  padding: 30px 30px;
  display: flex;
  flex-direction: column;
}

/* Title styles */
.title {
  width: 100%;
}

.title h1 {
  color: #45f3ff;
  justify-content: center;
  font-size: 2rem;
  font-weight: 600;
  opacity: 0.9;
  margin-bottom: 20px;
}

/* Input box styles */
form .input-box {
  width: 100%;
  margin-top: 20px;
}

form .input-box input {
  width: 100%;
  background-color: rgba(255, 255, 255, 0.2);
  border: none;
  outline: none;
  border-width: 3px;
  border-radius: 15px;
  padding: 10px 20px;
  font-size: 1rem;
  margin: 10px 0px 10px 0px;
  color: white;
}

form .input-box input::placeholder {
  color: #cdd1d2;
}

/* Submit button styles */
form .input-box input[type="submit"] {
  background-color: #45f3ff;
  cursor: pointer;
  color: #16100e;
  filter: drop-shadow(0 5px 10px #45f3ff);
  margin-bottom: 20px;
}

/* Link text styles */
form .link-text {
  padding-top: 15px;
  color: rgb(103, 173, 183);
  font-size: 0.85rem;
}

form .link-text a {
  text-decoration: none;
  color: rgb(153, 41, 99);
  font-weight: 700;
}

/* Label color */
.label-color {
  color: #9eb3b5;
}
<!DOCTYPE html>
<html>
  <head>
    <title>login page!</title>
    <link rel="stylesheet" href="styles.css" />
  </head>
  <body>
    <!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="UTF-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
    <title>Login Form</title>
    
    <link rel="stylesheet" href="css/style.css" />
  </head>
  <body>
    
    
    <div class="box">
      <form>
        
        <div class="title">
          <h1>Login Form</h1>
        </div>
        
        <div class="input-box">
          
          <label for="text" class="label-color">User name</label>
          <input
            id="text"
            name="text"
            type="text"
            placeholder="Username"
            required
          />
          <br />
          
          <label for="password" class="label-color">Password</label>
          <input
            id="password"
            name="password"
            type="password"
            placeholder="Password"
            required
          />
          <br />
          
          <input type="submit" class="Login" value="Login" />
          <br />
          
          <p class="link-text">Forget password ? <a href="#">Click Here</a></p>
          
          <p class="link-text">
            Dont have an account ?
            <a href="#">Click Here</a>
          </p>
        </div>
      </form>
    </div>
  </body>
</html>

     
      <p id="currentTime"></p>
      <script src="script.js"></script>
      <head>
    <meta charset="UTF-8" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
    <link rel="stylesheet" type="text/css" href="p.css" /> 
    <title>Link CSS to HTML</title>
   <style>
body, html {
  height: 100%;
  margin: 0;
}

.bg {
  /* The image used */
  background-image: url("f.jpg");

  /* Full height */
  height: 100%; 

  /* Center and scale the image nicely */
  background-position: center;
  background-repeat: no-repeat;
  background-size: cover;
}
</style>
</head>
 
  </body>
</html>

`
